lib/collection/src/shards/resolve.rs

```diff
@@ -208,9 +202,14 @@ where
 #[derive(Debug)]
 struct ResolverRecord<'a, T> {
     item: Option<&'a T>,
+    /// Offset within all results
     row: usize,
+    /// Offset within a set of results
     index: usize,
+    /// Keeps track of the amount of times we see this same item
     count: usize,
+    /// Keeps track of the index of the same item in different results. The lower, the earlier.
+    precedence: usize,
```

Comment:
How about just calling it order? Or do you want to prevent confusing it with order by?

Reply:
I mean it is not exactly order, it is just a heuristic for how early it should appear. It is the sum of the positions in each response

Reply:
Outdated :)